crypto/rand/rand_egd.c

```diff
@@ -61,14 +28,16 @@ int RAND_query_egd_bytes(const char *path, unsigned char *buf, int bytes)
 
 int RAND_egd(const char *path)
 {
-    return -1;
+    return (-1);
```

Comment:
I don't quite follow. Styling guide suggests no parenthesis... And it was fixed other way countless times...